### PR TITLE
4.0.2 release notes [main branch]

### PR DESCRIPTION
**Pull request by DavoudEshtehari**
**Todo**: updating release date.

### CODE REVIEW

**Review by lcheunglci — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -119,6 +119,22 @@ This update brings the below changes over the previous release:
 
 - Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1419](https://github.com/dotnet/SqlClient/pull/1419) [#1425](https://github.com/dotnet/SqlClient/pull/1425)
 
+## [Stable release 4.0.2] - 2022-09-09
+
+### Fixed
+
+- Fixed connection failure by skipping Certificate Revocation List (CRL) check during authentication. [#1718](https://github.com/dotnet/SqlClient/pull/1718)
```
Comment (David-Engel) [inline CHANGELOG.md:126]:
```suggestion
- Fixed connection failure by not requiring Certificate Revocation List (CRL) check during authentication. [#1718](https://github.com/dotnet/SqlClient/pull/1718)
```

File: release-notes/4.0/4.0.2.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 4.0.2 released 9 September 2022
+
+This update brings the below changes over the previous preview release:
+
+### Fixed
+
+- Fixed connection failure by skipping Certificate Revocation List (CRL) check during authentication. [#1718](https://github.com/dotnet/SqlClient/pull/1718)
```
Comment (David-Engel) [inline release-notes/4.0/4.0.2.md:9]:
```suggestion
- Fixed connection failure by not requiring Certificate Revocation List (CRL) check during authentication. [#1718](https://github.com/dotnet/SqlClient/pull/1718)
```

**Review by David-Engel — COMMENTED** [review]
We should mention the SNI crash fix (# 1418). I realize the fix was in the SNI code, but this is the only place that information would surface.

**Review by David-Engel — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -119,6 +119,22 @@ This update brings the below changes over the previous release:
...
+## [Stable release 4.0.2] - 2022-09-09
+
+### Fixed
+
+- Fixed connection failure by not requiring Certificate Revocation List (CRL) check during authentication. [#1718](https://github.com/dotnet/SqlClient/pull/1718)
+- Parallelize SSRP requests on Linux and macOS when MultiSubNetFailover is specified. [#1720](https://github.com/dotnet/SqlClient/pull/1720), [#1747](https://github.com/dotnet/SqlClient/pull/)
+- Added CommandText length validation when using stored procedure command types. [#1721](https://github.com/dotnet/SqlClient/pull/1721)
+- Fixed NullReferenceException during Azure Active Directory authentication. [#1722](https://github.com/dotnet/SqlClient/pull/1722)
+- Fixed null SqlBinary as rowversion. [#1724](https://github.com/dotnet/SqlClient/pull/1724)
+- Fixed table's collation overriding with default UTF8 collation. [1750](https://github.com/dotnet/SqlClient/pull/1750)
```
Comment (lcheunglci) [inline CHANGELOG.md:131]:
```suggestion
- Fixed table's collation overriding with default UTF8 collation. [#1750](https://github.com/dotnet/SqlClient/pull/1750)
```

File: release-notes/4.0/4.0.2.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+This update brings the below changes over the previous preview release:
+
+### Fixed
+
+- Fixed connection failure by not requiring Certificate Revocation List (CRL) check during authentication. [#1718](https://github.com/dotnet/SqlClient/pull/1718)
+- Parallelize SSRP requests on Linux and macOS when MultiSubNetFailover is specified. [#1720](https://github.com/dotnet/SqlClient/pull/1720), [#1747](https://github.com/dotnet/SqlClient/pull/)
+- Added CommandText length validation when using stored procedure command types. [#1721](https://github.com/dotnet/SqlClient/pull/1721)
+- Fixed NullReferenceException during Azure Active Directory authentication. [#1722](https://github.com/dotnet/SqlClient/pull/1722)
+- Fixed null SqlBinary as rowversion. [#1724](https://github.com/dotnet/SqlClient/pull/1724)
+- Fixed table's collation overriding with default UTF8 collation. [1750](https://github.com/dotnet/SqlClient/pull/1750)
```
Comment (lcheunglci) [inline release-notes/4.0/4.0.2.md:14]:
```suggestion
- Fixed table's collation overriding with default UTF8 collation. [#1750](https://github.com/dotnet/SqlClient/pull/1750)
```

**Pull request merged**